pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -754,6 +769,10 @@ func doLedAction(ledName string, turnon bool) {
 // doLedBlink can use different LEDs in /sys/class/leds
 // Enable the led for 200ms
 func doLedBlink(ledName string) {
+	if ledName == "" {
+		time.Sleep(200 * time.Millisecond)
```

Comment:
I got the point here. We don't want to mess up time aware functions if some LED doesn't exist, right? Well, but at the other hand it can mask the problem in case some LED doesn't exist to the callers. Maybe would be better to explicitly put this delay in the callers in case of error, instead let "hidden" here....